pkgs/applications/video/gnome-subtitles/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, fetchFromGitLab
+, autoreconfHook
+, pkg-config
+, intltool
+, itstool
+, mono
+, yelp-tools
+, wrapGAppsHook
+, enchant1
+, glib
+, gnome3
+, gtk-sharp-3_0
+, gst_all_1
+, libxml2
+}:
+let
+  version = "1.6";
+in
+stdenv.mkDerivation {
+  pname = "gnome-subtitles";
+  inherit version;
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gnome-subtitles";
+    rev = "gnome-subtitles-${version}";
+    sha256 = "1crmmcx32i6ca7dlr3xhnc7vgv9jhlpwh6hxhv2fl1x1zbasf42z";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    autoreconfHook
+    mono
+    intltool
+    itstool
+    pkg-config
+    yelp-tools
+  ];
+
+  buildInputs = [
+    enchant1
```

Comment:
Enchant 1 has been unmaintained for a long time. Cannot it use Enchant 2?

Reply:
Unfortunately, it crashes when using Enchant2. It looks after libenchant.so.1 and is not finding it.

Reply:
Is there an issue for that upstream? I want to remove enchant1 before the end of the year.

Reply:
It looks like the code in master branch supports enchant2 out-of-the box but the 1.6 brnach does not. So I did a substituteInPlace for the library name